tap_hubspot/__init__.py

```diff
@@ -189,7 +191,8 @@ def load_associated_company_schema():
     return associated_company_schema
 
 def load_schema(entity_name):
-    schema = utils.load_json(get_abs_path('schemas/{}.json'.format(entity_name)))
+    entity_class = entity_name.split(":")[0]
```

Comment:
is this change needed if line 940 does the same thing or does it fix other entities?

Reply:
The tricky part is when we discover streams (please, refer to the line 1087). If it was only for loading a schema during the data import I wouldn't implement it here. So there's a decision to be made: either we introduce `stream_class` as an attribute of the `Stream` or we parse `stream_id` here. The latter seems nicer and cleaner to me. Your take?